<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>541</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>280</y>
      <width>273</width>
      <height>156</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QPushButton" name="btnCameraPanLeft">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/img/rotate-left.png</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>45</width>
         <height>45</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnCameraTiltUp">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="projectressources.qrc">
         <normaloff>:/img/arrow-up-bold-box-outline.png</normaloff>
         <normalon>:/img/arrow-up-bold-box.png</normalon>:/img/arrow-up-bold-box-outline.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>45</width>
         <height>45</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btnCameraTiltDown">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="projectressources.qrc">
         <normaloff>:/img/arrow-down-bold-box-outline.png</normaloff>
         <normalon>:/img/arrow-down-bold-box.png</normalon>:/img/arrow-down-bold-box-outline.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>45</width>
         <height>45</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btnCameraPanRight">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/img/rotate-right.png</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>45</width>
         <height>45</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="lblCameraPartTitle">
       <property name="text">
        <string>Camera</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>30</y>
      <width>351</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Camera Settings</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>351</width>
          <height>171</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="frameLayout"/>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lblServerAddress">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Robot Address</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>373</width>
      <height>188</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0" rowspan="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QSlider" name="sliderSpeed">
       <property name="maximum">
        <number>280</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblMoveSpeed">
       <property name="text">
        <string>Speed</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2" alignment="Qt::AlignVCenter">
      <widget class="QPushButton" name="btnMoveForward">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="projectressources.qrc">
         <normaloff>:/img/arrow-up-bold-box-outline.png</normaloff>
         <normalon>:/img/arrow-up-bold-box.png</normalon>:/img/arrow-up-bold-box-outline.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>45</width>
         <height>45</height>
        </size>
       </property>
       <property name="shortcut">
        <string notr="true">Up</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2" alignment="Qt::AlignBottom">
      <widget class="QPushButton" name="btnMoveBackward">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/img/arrow-down-bold-box-outline.png</normalon>
         <selectedon>:/img/arrow-down-bold-box.png</selectedon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>45</width>
         <height>45</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Down</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btnMoveRotateLeft">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/img/rotate-left.png</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>45</width>
         <height>45</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Left</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="btnMoveRotateRight">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/img/rotate-right.png</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>45</width>
         <height>45</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Right</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLabel" name="lblMovePartTitle">
       <property name="text">
        <string>Moves</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1089</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>Connection</string>
    </property>
    <addaction name="actionConnectTo"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <addaction name="menuConnection"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionConnectTo">
   <property name="text">
    <string>Connect to</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="projectressources.qrc"/>
 </resources>
 <connections/>
</ui>
